### PR TITLE
New: ICBC.ICBCEBankAssist version 1.9.3.0

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27270)

### CODE REVIEW

File: manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+PackageIdentifier: ICBC.ICBCEBankAssist
+PackageVersion: 1.9.3.0
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: http://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/x64/ICBCSetupIntegration_64.msi
+  InstallerSha256: DC6813ED931207CD1192A385A6C9E5BEDFDC9A05F5A75F0781F91E5A778E624A
```
Comment (ItzLevvie) [inline manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml:12]:
```suggestion
  InstallerSha256: DC6813ED931207CD1192A385A6C9E5BEDFDC9A05F5A75F0781F91E5A778E624A
  ProductCode: "{292FC34F-1862-4630-9AD6-D45079053A10}"
```

File: manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  UpgradeBehavior: install
+- Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/ICBCSetupIntegration.msi
+  InstallerSha256: FD3D61AF313043736D5F4C3AAD56E86CEBD180064E83413DAF59E8EA90E49C51
```
Comment (ItzLevvie) [inline manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml:18]:
```suggestion
  InstallerSha256: FD3D61AF313043736D5F4C3AAD56E86CEBD180064E83413DAF59E8EA90E49C51
  ProductCode: "{7ECFFD48-0558-44DD-B9E2-22F7DCD1A203}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/i/ICBC/ICBCEBankAssist/1.9.3.0/ICBC.ICBCEBankAssist.installer.yaml:12]: I got a different ProductCode "{7A712B0E-175E-435C-BC4A-793DC903C23C}" after installing x64 version.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You might probably have to do something like this (added InstallerLocale because it seems like installing the application on other system languages gives you different product codes)

Replace "<Enter ProductCode for zh-CN ... locale here>" with the ProductCode you got when installing the application.

```
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: msi
  Scope: machine
  InstallerUrl: http://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/x64/ICBCSetupIntegration_64.msi
  InstallerSha256: DC6813ED931207CD1192A385A6C9E5BEDFDC9A05F5A75F0781F91E5A778E624A
  ProductCode: "{292FC34F-1862-4630-9AD6-D45079053A10}"
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/ICBCSetupIntegration.msi
  InstallerSha256: FD3D61AF313043736D5F4C3AAD56E86CEBD180064E83413DAF59E8EA90E49C51
  UpgradeBehavior: install
  ProductCode: "{7ECFFD48-0558-44DD-B9E2-22F7DCD1A203}"
- InstallerLocale: zh-CN
  Architecture: x64
  InstallerType: msi
  Scope: machine
  InstallerUrl: http://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/x64/ICBCSetupIntegration_64.msi
  InstallerSha256: DC6813ED931207CD1192A385A6C9E5BEDFDC9A05F5A75F0781F91E5A778E624A
  ProductCode: "<Enter ProductCode for zh-CN x64 locale here>"
- InstallerLocale: zh-CN
  Architecture: x86
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/ICBCSetupIntegration.msi
  InstallerSha256: FD3D61AF313043736D5F4C3AAD56E86CEBD180064E83413DAF59E8EA90E49C51
  UpgradeBehavior: install
  ProductCode: "<Enter ProductCode for zh-CN x86 locale here>"
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**